pkg/core/interop/context.go

```diff
@@ -153,6 +153,9 @@ type MethodAndPrice struct {
 // Contract is an interface for all native contracts.
 type Contract interface {
 	Initialize(*Context) error
+	// ActiveIn returns the hardfork native contract is active from or nil in case
+	// it's always active.
+	ActiveIn() *config.Hardfork
```

Comment:
I'm not sure what #3213 will look like in the end, it's likely to affect this interface since we might have different versions at different heights while currently they're _all_ enabled at 0 and never change. But not critical for now.